pkg/channel/distributed/controller/kafkachannel/kafkachannel.go

```diff
@@ -49,6 +51,9 @@ func (r *Reconciler) reconcileKafkaChannel(ctx context.Context, channel *kafkav1
 // Reconcile The KafkaChannels MetaData (Annotations, Labels, etc...)
 func (r *Reconciler) reconcileMetaData(ctx context.Context, channel *kafkav1beta1.KafkaChannel) error {
 
+	// Get The Logger From The Context
+	logger := logging.FromContext(ctx)
+
```

Comment:
Should this be a "wrapped" ChannelLogger as is done above in `reconcileKafkaChannel` ?

Reply:
That's a good idea; I'll see if it works

Reply:
It worked pretty well; I've moved the util.ChannelLogger call as far up the stack as is reasonable.